src/conv/k-g/extrude.cpp

```diff
@@ -0,0 +1,87 @@
+/*                 E X T R U D E . C P P
+ * BRL-CAD
+ *
+ * Copyright (c) 2024 United States Government as represented by
+ * the U.S. Army Research Laboratory.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public License
+ * version 2.1 as published by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this file; see the file named COPYING for more
+ * information.
+ */
+ /** @file extrude.cpp
+  *
+  * LS Dyna keyword file to BRL-CAD converter:
+  * intermediate extrude implementation
+  */
+
+#include "extrude.h"
+
+
+
+
+Extrude::Extrude(void)
+{
+    BU_GET(m_extrude, rt_extrude_internal);
+    m_extrude->magic = RT_EXTRUDE_INTERNAL_MAGIC;
+}
+
+
+void Extrude::setName
+(
+    const char* value
+) {
+    if (value != nullptr)
+	name = value;
+    else
+	name = "";
+}
+
+
+void Extrude::extrudeSection(std::string sectionName,const point_t& V, vect_t h, vect_t u_vec, vect_t v_vec, rt_sketch_internal* skt)
+{
```

Comment:
I get the following compilation errors here:
```[ 79%] Building CXX object src/conv/k-g/CMakeFiles/k-g.dir/extrude.cpp.o
/home/rossberg/Devel/BRL-CAD/brlcad.devel_k-g/src/conv/k-g/extrude.cpp: In member function ‘void Extrude::extrudeSection(std::string, const fastf_t (&)[3], fastf_t*, fastf_t*, fastf_t*, rt_sketch_internal*)’:
/home/rossberg/Devel/BRL-CAD/brlcad.devel_k-g/src/conv/k-g/extrude.cpp:49:130: error: unused parameter ‘skt’ [-Werror=unused-parameter]
   49 | on(std::string sectionName,const point_t& V, vect_t h, vect_t u_vec, vect_t v_vec, rt_sketch_internal* skt)
      |                                                                                    ~~~~~~~~~~~~~~~~~~~~^~~
```

Reply:
yes, I though we might need it for the extrude. I have to remove it